File: Day-068/Array.js
// DO NOT USE ARGUMENTS FOR INPUTS
// Vaiable size_ is the size of input array.
// Variable array_ stores all the array elements
var size_ = parseInt(readLine().trim());
var array_ = []
for(var i = 0 ; i < size_; i++ ){
    var x = parseInt(readLine().trim());
    array_.push(x);
}
// YOUR CODE GOES HERE
var new_arr = [];  


for (var i = 0; i < array_.length; i++) {


    // push items based in old array + i 
    new_arr.push(array_[i] + i);  

} 

// Based on New array's length - print each new array's element
for (var i = 0; i < new_arr.length; i++) {

    console.log(new_arr[i]);
}

File: Day-108/Array.cpp
/*
#include<iostream>
using namespace std;
*/

void reverseArray(int arr[], int N){
    // Your code goes here
    int temp;
    for (int i = 0 ; i < N/2 ; i++){
        temp = arr[i];
        arr[i] = arr[N-1-i];
        arr[N-1-i] = temp;
    }
}


/*
int main()  {
    int N;
    cin>>N;
    int arr[N];
    for(int i = 0; i < N; i++){
        cin>>arr[i];
    }
    reverseArray(arr, N);
    for(int i = 0; i < N; i++){
        cout<<arr[i]<<" ";
    }
    cout<<endl;
    return 0;
}
*/
